project/sites/pressclippingsnics/settings.php

```diff
@@ -6,3 +6,18 @@
 $subsite_id = basename(__DIR__);
 
 include $app_root . '/sites/site.settings.php';
+
+/**
+ * Private file path:
+ *
+ * A local file system path where private files will be stored. This directory
+ * must be absolute, outside of the Drupal installation directory and not
+ * accessible over the web.
+ *
+ * Note: Caches need to be cleared when this value is changed to make the
+ * private:// stream wrapper available to the system.
+ *
+ * See https://www.drupal.org/documentation/modules/file for more information
+ * about securing private files.
+ */
+$settings['file_private_path'] = $app_root . '/web/files/pressclippingsnics/privatefiles';
```

Comment:
Not sure this path is correct, I would expect it to be '/files/pressclippingsnics/privatefiles'